ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml.cs

```diff
@@ -185,6 +199,13 @@ protected override async Task ValidateAndSave()
             }
         }
 
+        private bool HasNonPostMortemTreatmentEvents()
+        {
+            return Notification.TreatmentEvents
+                .Any(te => te.TreatmentEventType != TreatmentEventType.DiagnosisMade
+                           && (!te.TreatmentEventTypeIsOutcome || te.TreatmentOutcome.TreatmentOutcomeType != TreatmentOutcomeType.Died));
```

Comment:
Would it be worth adding a property to a treatment event called something like `IsDiedOutcome`? It would make this more readable, and we use the same logic elsewhere

Reply:
Turns out this property exists, and was just private. Have made it public and am using it now.